### PR TITLE
catalog page and updates

### PR DESCRIPTION
**Pull request by IvayloSlavchev**

### CODE REVIEW

File: client/src/components/Header/Catalog/Catalog.js [hunk 1]
```diff
@@ -1,11 +1,55 @@
...
+
+                        <Button className='catalog-btns firstButtonWidth'>
+                            <HomeOutlinedIcon style={{ fontSize: '1.5rem' }} />
+                            <Typography variant='paragraph' className='catalog-typography'>Main page</Typography>
+                            <ArrowForwardIosIcon style={{ fontSize: '.7rem' }} />
+                            <Typography variant='paragraph' className='catalog-typography'>Catalog</Typography>
+                        </Button>
+                    </Grid>
+                    <Grid xl={2.77}>
+                        <Button className='catalog-btns secondButtonWidth'>
```
Comment (petya0111) [inline client/src/components/Header/Catalog/Catalog.js]: let's have more descriptive names and not 'firstButtonWidth', 'secondButtonWidth' ..

File: client/src/components/Header/Catalog/SideBar.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                                            }
+                                        </RadioGroup>
+                                    </FormControl>
+                                </Typography>
+                            </AccordionDetails>
+
+                        </Accordion>
+                    </Box>
+
+                    <Box className='side-bar-boxes'>
```
Comment (petya0111) [inline client/src/components/Header/Catalog/SideBar.js]: try to separate this  <Box className='side-bar-boxes'> in separate component to not have repeated code 
and loop needed data like this example here https://stackoverflow.com/a/22877110

File: client/src/components/Header/Catalog/Catalog.js (hunk 1 above)
Comment (IvayloSlavchev) [reply client/src/components/Header/Catalog/Catalog.js]: Fixed

File: client/src/components/Header/Catalog/SideBar.js (hunk 2 above)
Comment (IvayloSlavchev) [reply client/src/components/Header/Catalog/SideBar.js]: Fixed

**Review by Arabadzhiew — APPROVED** [review]

**Review by JordanRad — APPROVED** [review]

**Review by petya0111 — APPROVED** [review]

**Comment by petya0111** [comment]
@IvayloSlavchev  try to use Material UI Grid functionality to achieve a responsive sidebar and navbar :)

**Review by petya0111 — COMMENTED** [review]
.

**Review by petya0111 — CHANGES_REQUESTED** [review]
@IvayloSlavchev  try to use Material UI Grid functionality to achieve a responsive sidebar and navbar :)

**Pull request merged**